api/net/Program.cs

```diff
@@ -187,6 +187,7 @@
     options.SubstituteApiVersionInUrl = true;
 
 });
+builder.Services.AddMemoryCache();
```

Comment:
Not using this yet, but we can add this later to improve performance.  But I'm incline to move things to Redis before doing much with this.